content/usage/basic.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Basic command examples"
+weight: 1
+---
+
+Here will go some basic examples of using TinyGo in the most common scenarios.
+
+### Building "Hello, World" program for WebAssembly
+
+To build the WASM example, run the following command:
+
+```shell
+tinygo build -o wasm.wasm -target=wasm examples/wasm
+```
+
+OK, so what do you do now? Some details from the community here please.
```

Comment:
For example:

> If you have Emscripten installed, you can run a local webserver inside the TinyGo root:
> 
>     emrun --no_browser .
> 
> Then you can open http://localhost:6931/src/examples/wasm/wasm.html and see your compiled WebAssembly file in action.

Reply:
See the PR submitted by @johanbrandhorst which is a more complete set of info about WASM: https://github.com/tinygo-org/tinygo-site/pull/13 

I'd prefer to not send the developer down the path of figuring out how to install Emscripten.

Reply:
Yes, that documentation is a lot better.